app/controllers/api/base.rb

```diff
@@ -1,7 +1,19 @@
 # frozen_string_literal: true
 
+require 'doorkeeper/grape/helpers'
+
 module API
   class Base < Grape::API
+    helpers Doorkeeper::Grape::Helpers
+
+      before do
+        doorkeeper_authorize!
```

Comment:
not sure if we need it here, not all the endpoints need user to be authorized already
Please, make sure, if endpoint without jwt in headers will not crash like this. thanks

Reply:
@andreymakovenko I think, Admin should be authorized anyway in order to access admin panel.

Reply:
crap, I forgot it's **admin** API :smile:

Reply:
@gfedorenko yes, and this should go into separate admin controller, not in the base one